numerics/polynomial_evaluators_body.hpp

```diff
@@ -323,6 +323,7 @@ Derivative<Value, Argument>
 HornerEvaluator<Value, Argument, degree>::EvaluateDerivative(
     Coefficients const& coefficients,
     Argument const& argument) {
+  // Starting at 1 prevents us from having polynomials of degree 0.
```

Comment:
Should we special-case this and return 0?

Reply:
We could, although it's hard to be excited about degree 0.  Maybe in a future PR...